CoreVulkanDisplay/src/au/gov/asd/tac/constellation/visual/vulkan/CVKVisualProcessor.java

```diff
@@ -68,19 +68,25 @@
     
     private final BlockingQueue<CVKRenderable.CVKRenderableUpdateTask> taskQueue = new LinkedBlockingQueue<>();
     private final CVKCanvas cvkCanvas;
-    private final CVKRenderer cvkRenderer;
     private final Frustum viewFrustum = new Frustum();
     private final Matrix44f projectionMatrix = new Matrix44f();       
-    private CVKIconTextureAtlas cvkIconTextureAtlas = null;
-    private CVKAxesRenderable cvkAxes = null;
-    private CVKFPSRenderable cvkFPS = null;
-    private CVKIconsRenderable cvkIcons = null;
-    private CVKPointsRenderable cvkPoints = null;
+    private final CVKIconTextureAtlas cvkIconTextureAtlas;
+    protected final CVKHitTester cvkHitTester;
+    private final CVKAxesRenderable cvkAxes;
+    private final CVKFPSRenderable cvkFPS ;
+    private final CVKIconsRenderable cvkIcons;
+    private final CVKPointsRenderable cvkPoints;
     private final Matrix44f modelViewMatrix = new Matrix44f();  
-    private Camera camera;
+    private Camera camera = null;
     private float pixelDensity = 0.0f;    
     private final Thread renderThread;
-    public final CVKGraphLogger cvkLogger;
+    protected final CVKGraphLogger cvkLogger;
+    
+    
+    // TODO: how should this work with continuous rendering?
```

Comment:
I would think it is ok for the hit tester to update continuously as well. It isn't efficient but the behaviour is the same in both cases at least which is good for debugging